app/controllers/api/auth/stories_controller.rb

```diff
@@ -1,14 +1,17 @@
 # encoding: utf-8
 
 class Api::Auth::StoriesController < Api::StoriesController
-
   include ApiAuthenticated
 
   api_versions :v1
 
   filter_resources_by :account_id, :series_id, :network_id
 
-  filter_params :noseries
+  filter_params :highlighted, :purchased, :v4, :text, :noseries
+
+  sort_params default: { updated_at: :desc },
```

Comment:
no, when I call `sort_params`, I have to specify both, so line 13 has the allowed sort params.
(I didn't implement partial overrides.)